testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/SlaveHostControllerAuthenticationTestCase.java

```diff
@@ -58,21 +31,29 @@
 import org.jboss.as.test.integration.security.common.VaultHandler;
 import org.jboss.dmr.ModelNode;
 import org.junit.AfterClass;
-import org.junit.Assert;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
+import javax.security.auth.callback.CallbackHandler;
```

Comment:
Sorry, fixed my IDE setup. The ordering should be as in the rest of the project now.